server/controllers/userController.js

```diff
@@ -20,13 +23,58 @@ export default class UserController {
   */
   static create(req, res) {
     const { username, email, password } = req.body;
+
+    // generate verification token and store in verifyToken variable
+    const verifyToken = crypto.randomBytes(16).toString('hex');
+
     bcrypt.hash(password, saltRounds)
-      .then(hashedPassword => User.create({ username, email, password: hashedPassword, })
+      .then(hashedPassword => User.create({
+        username, email, password: hashedPassword, verifyToken
+      })
         .then((user) => {
           const { id } = user;
           const token = generateToken({ id });
-          res.status(201).json({ success: true, message: 'You have successfully signed up!', token });
```

Comment:
Change message to 'User successfully signed up, an email is sent to your mail account, please verify your mail account to complete registration' to inform the user to check the mail and verify their account.